shimming-toolbox/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:latest
```

Comment:
Bad idea to get the latest version as an update could create issues without even changing the code. A fixed version would make the Docker image more stable.